src/aap_eda/core/models/rulebook_process.py

```diff
@@ -196,3 +196,29 @@ class Meta:
     line_number = models.IntegerField()
     log = models.TextField()
     log_timestamp = models.BigIntegerField(null=False, default=0)
+
+
+class RulebookProcessQueue(models.Model):
+    """Rulebook Process Queue model.
+
+    Rulebook Process Queue is an instance of ansible-rulebook process
+    that is created when a rulebook process is started.
+    It keeps track of the queue name where the process is running.
+    """
+
+    queue_name = models.CharField(max_length=255)
```

Comment:
@Alex-Izquierdo Wouldn't this be better as a separate Node object that can have a relationship to one or more Rulebook Processes. The Node object can have a Queue Name/Host Name/Uptime/..... and can be updated separately and when a failure id detected on the Node flag all the running rulebook processes.

Reply:
Right now we are not managing nodes, is out of the scope, queues (nodes) are provided through static settings and handled at runtime. Node management can be done in the next release, it would require explicit tools for managing them (register/deregister comands) as well as dynamic queues.